internal/services/network/network_packet_capture_resource.go

```diff
@@ -231,6 +281,7 @@ func resourceNetworkPacketCaptureRead(d *pluginsdk.ResourceData, meta interface{
 
 	if props := resp.PacketCaptureResultProperties; props != nil {
 		d.Set("target_resource_id", props.Target)
+		d.Set("target_type", props.TargetType)
```

Comment:
This will need to be defaulted since it won't be returned unless it's been explicitly set previously

Reply:
I tried. When it isn't set in request body, API would return default value. So I think we don't need to process it specially.

Reply:
This is true for newly created resources, but doesn't apply to already existing ones created using an older API version, so this will still need to be defaulted.

Reply:
Thanks. Updated